tests/chainer_tests/dataset_tests/tabular_tests/test_concat.py

```diff
@@ -0,0 +1,95 @@
+from operator import xor
```

Comment:
(style) Import module instead of its items (do `import operator` and use `operator.xor`).